pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[build-system]
+requires = [
+    "wheel",
+    "setuptools",
+    "Cython>=0.29.18",
+    "numpy>=1.15.1",
```

Comment:
I believe that doing this will almost surely cause numpy 1.19 to be installed at build time, which would make it incompatible with older versions of numpy which the user may have installed in their environment.

This is why scipy had the version pinned at build time.

See conda-forge's knowledge base on this:
https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
This is really great of you. Thank you for being so helpful and having these links handy.

Reply:
I've put it back to the simple fix suggested earlier.